_posts/2016-03-02-Rust-1.7.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: "Announcing Rust 1.7"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.7. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.7][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.7][notes] on GitHub.
+About 1300 patches were landed in this release.
+
+[install]: http://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-170-2016-03-03
+
+### What's in 1.7 stable
+
+This release is primarily about library features. While we have several
+language features cooking for future releases, the timeframe in which 1.7
+included the holidays, which means less time for commenting on GitHub and more
+time for spending with loved ones.
+
+#### New Language features
+
+`None`
+
+#### Library stabilizations
+
+About 40 library functions and methods are now stable in 1.7. Notable
+improvements include:
+
+* The ability to customize `HashMap<K, V>`s and similar data structures with
```

Comment:
In particular, this is a good opportunity to talk about how our hashing design differs from the simple story in e.g. Java, and what the implications are.